app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,23 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        let resendEmail = await this.loader.post('orders/resend-email', payload);
+        if (resendEmail.status) {
+          this.notify.success(this.l10n.t('Email confirmation has been sent to attendess successfully'));
```

Comment:
@uds5501 `Attendees` I referred to the following typo before 😄

Reply:
I prefer if we stick to error message used across the app 
`An unexpected error occurred.`